Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -77,7 +78,7 @@
 - I invest in sustainable delivery, maintaining professional standards and striking a pragmatic balance between idealistic purity and business pressures.
 - I understand our fixed-time/high-quality/variable-scope approach to software projects.
 - I assess what work supports the product strategy and make insightful recommendations regarding priorities.
-- I identify key gaps in product offerings and functionality that will drive significant revenue and customer excitement.
+- I identify key gaps in product offerings and functionality, then work with a Product Manager to shape the opportunities.
```

Comment:
Could we apply a similar "role agnostic" slant to this one too, e.g. if it was working with members of leadership

> I identify key gaps in product offerings and functionality, then work with **responsible team members (e.g. Product Managers, Business Stakeholders)** to shape the opportunities.